lib/flay.rb

```diff
@@ -170,7 +170,7 @@ def self.load_plugins
   # Create a new instance of Flay with +option+s.
 
   def initialize option = nil
-    @option = option || Flay.default_options
+    @option = option || self.class.default_options
```

Comment:
if Fooflay overrides Flay and defines overrides default_options, then these options should be used and not the original Flay options ... atm it makes overriding do nothing